pkg/probe/turbo_target_info.go

```diff
@@ -0,0 +1,76 @@
+package probe
+
+import (
+	"github.com/turbonomic/turbo-api/pkg/api"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+)
+
+// Abstraction for the TurboTarget object in the client
+type TurboTargetInfo struct {
+	// Category of the target, such as Hypervisor, Storage, etc
+	targetCategory string
```

Comment:
We should keep the same names 'ProbeCategory' and 'ProbeType' instead of new targetCategory and targetType.